src/TaglibSharp.Tests/Images/BadIptcSectionTest.cs

```diff
@@ -0,0 +1,102 @@
+using NUnit.Framework;
+using NUnit.Framework;
+using TagLib;
+using TagLib.IFD;
+using TagLib.IFD.Tags;
+using TagLib.IFD.Entries;
+using TagLib.Jpeg;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaglibSharp.Tests.Images.Validators;
+using TagLib.IFD;
+using TagLib.Xmp;
```

Comment:
Can you sort these with System at the top. Also, there are a couple duplicates in this list.

Reply:
If you can fix this, I'm good with the changes and will merge them.

Reply:
Actually, none of the System using statements were needed according to VS 2022, so I removed them along with the other redundant or otherwise unnecessary using statements.